### ISSUE TITLE
Missing quadrotor_msgs package?

### ISSUE THREAD

**Issue by juanmed** [uzh-rpg/vimo#2]
Hi,

Thanks for sharing this interesting research implementation. Trying to build your code, I get the an error indicating a missing `quadrotor_msgs` dependency:

```bash
==> add_subdirectory(vimo/vimo_estimator)
-- Using these message generators: gencpp;geneus;genlisp;gennodejs;genpy
-- Could NOT find quadrotor_msgs (missing: quadrotor_msgs_DIR)
-- Could not find the required component 'quadrotor_msgs'. The following CMake error indicates that you either need to install the package with the same name or change your environment so that it can be found.
CMake Error at /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):
  Could not find a package configuration file provided by "quadrotor_msgs"
  with any of the following names:

    quadrotor_msgsConfig.cmake
    quadrotor_msgs-config.cmake

  Add the installation prefix of "quadrotor_msgs" to CMAKE_PREFIX_PATH or set
  "quadrotor_msgs_DIR" to a directory containing one of the above files.  If
  "quadrotor_msgs" provides a separate development package or SDK, be sure it
  has been installed.
```

Indeed, in `vimo_estimator` package's CMakeLists file the dependency is included, but there is no actual package defining that message. Could you please let me know where is that dependency or what are the fields of the `quadrotor_msgs` message so that I can define it by myself?

Thanks,

**Comment by juanmed** [comment]
I might have found the issue, this repository is dependent also on the following:

https://github.com/uzh-rpg/rpg_quadrotor_common
https://github.com/catkin/catkin_simple
https://github.com/ethz-asl/eigen_catkin

After cloning this into the workspace, the build process is successful, however I still have to make it run.

**Comment by barzanisar** [comment]
Hi, 
Thanks for pointing it out. I'll update the README with links to these repositories.

Blackbird branch does not need quadrotor_msgs and quadrotor_common dependencies. I removed them from its CMakeLists.
For the master, you need the above mentioned repositories in your workspace.

**Comment by juanmed** [comment]
Hi, thanks for the explanation. I am looking at it right now and can see those dependencies in the blackbird branch. Will pull your changes soon. I can do a pull request with the links to the repositories if you want.

However, when building the blackbird branch, I also got an error with the blackbird/MotorRPM message not being available. These are coming from the blackbird dataset, but I was wondering if I should build them together with your packages? if yes, if any location is preferred or required?

```
Errors     << vimo_estimator:make /home/fer/git_clone/vimo_ws/logs/vimo_estimator/build.make.000.log              
In file included from /home/fer/git_clone/vimo_ws/src/vimo/vimo_estimator/src/estimator_node.cpp:12:0:
/home/fer/git_clone/vimo_ws/src/vimo/vimo_estimator/src/estimator.h:13:32: fatal error: blackbird/MotorRPM.h: No such file or directory
In file included from /home/fer/git_clone/vimo_ws/src/vimo/vimo_estimator/src/estimator.cpp:1:0:
/home/fer/git_clone/vimo_ws/src/vimo/vimo_estimator/src/estimator.h:13:32: fatal error: blackbird/MotorRPM.h: No such file or directory
```

Thank you,

**Comment by juanmed** [comment]
I was able to run your code using the blackbird dataset and get the external force estimations. The only 2 bits information  not in the readme are 1) the missing dependencies and 2) the blackbird messages. Regarding the blackbird messages I simply created a new package called `blackbird` and created `MotorRPM` message type in a `msg/` folder following the structure in the blackbird dataset. I imagine there are other solutions.

Let me know if a pull request would be useful. If yes, how to handle the blackbird messages dependency.

Thanks,
